namelayer-spigot/src/main/java/vg/civcraft/mc/namelayer/database/GroupManagerDao.java

```diff
@@ -356,7 +357,17 @@ public void run() {
 					if (g == null) {
 						createGroup(NameLayerPlugin.getSpecialAdminGroup(), null, null);
 					} else {
-						removeAllMembers(g.getName());
+						try (Connection connection = db.getConnection();PreparedStatement removeAllMembers = connection.prepareStatement("delete fm.* from faction_member fm "
+								+ "inner join faction_id fi on fi.group_id = fm.group_id where fi.group_name =?;"))  {
+							removeAllMembers.setString(1, g.getName());
+							removeAllMembers.execute();
+						} catch (SQLException e) {
+							e.printStackTrace();
+						}
+						
+						
+						cleanExecute("delete fm.* from faction_member fm inner join groupPlayerTypes gpt on gpt.type_id = fm.type_id "
```

Comment:
It's always better to have a prepared statement that might be useful elsewhere wrapped in an accessor function. You lose nothing and gain much by leaving it that way -- literally why I made it a function a few commits ago.

the closing `cleanExecute` is malformed, is all (lacks param), and if it did run would just remove all members from all groups.